src/containers/MealsPage/MealsPage.js

```diff
@@ -0,0 +1,140 @@
+    calories: '187 Calories',
+    category: 'Breakfast',
+    duration: '2 min.',
+    id: 2,
+  },
+  {
+    image:
+      'https://i.pinimg.com/236x/57/57/5a/57575a79673cc0f0a0848bb5f328b588.jpg',
+    title: 'Kiwi, Greek yogurt & blueberries',
+    calories: '170 Calories',
+    category: 'Breakfast',
+    duration: '15 min.',
+    id: 3,
+  },
+  {
+    image:
+      'https://keyassets-p2.timeincuk.net/wp/prod/wp-content/uploads/sites/53/2019/01/chicken-casserole-recipe.jpg',
+    title: 'Classic Chicken Casserole',
+    calories: '390 Calories',
+    category: 'Lunch',
+    duration: '30 min.',
+    id: 4,
+  },
+  {
+    image:
+      'https://cdn.apartmenttherapy.info/image/upload/f_auto,q_auto:eco,w_730/k%2Farchive%2F985a73c50098583a0f87d6f5f28b8cced8ab2335',
+    title: 'Tuna Salad',
+    calories: '187 Calories',
+    category: 'Lunch',
+    duration: '10 min.',
+    id: 5,
+  },
+  {
+    image:
+      'https://cdn.apartmenttherapy.info/image/upload/v1559144163/k/archive/a54502124bcf08139f5c4a87b713c01564139869.jpg',
+    title: 'Turkey & Cheese Bites',
+    calories: '150 Calories',
+    category: 'Lunch',
+    duration: '20-25 min.',
+    id: 6,
+  },
+  {
+    image:
+      'https://keyassets-p2.timeincuk.net/wp/prod/wp-content/uploads/sites/53/2017/04/mexican-stuffed-chicken-with-red-rice-scaled.jpg',
+    title: 'Mexican Stuffed Chicken with red rice',
+    calories: '467 Calories',
+    category: 'Dinner',
+    duration: '25 min.',
+    id: 7,
+  },
+  {
+    image:
+      'https://www.bbcgoodfood.com/sites/default/files/styles/recipe/public/recipe_images/miso-roasted-aubergine-steaks.jpg',
+    title: 'Miso-roasted aubergine steaks with sweet potato',
+    calories: '344 Calories',
+    category: 'Dinner',
+    duration: '20 min.',
+    id: 8,
+  },
+  {
+    image:
+      'https://www.bbcgoodfood.com/sites/default/files/recipe_images/spaghetti_4.jpg',
+    title: 'Super smoky bacon & tomato spaghetti',
+    calories: '500 Calories',
+    category: 'Dinner',
+    duration: '25 min.',
+    id: 9,
+  },
+  {
+    image:
+      'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQdxxYX3kXKqal2R8J_RazipFc5xUjKRxQmFKeMyBkQ69TMGywV&s',
+    title: ' Sweet Potato Chips',
+    calories: '100 Calories',
+    category: 'Snacks',
+    duration: '2 hrs.',
+    id: 10,
+  },
+  {
+    image:
+      'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTn_-PAvNOU0xi-OC6QWgasq4HmYfxJavPXU7ukxM7wzx3F5I66&s',
+    title: '  Peanut Butter and Honey Oat Bar',
+    calories: '157 Calories',
+    category: 'Snacks',
+    duration: '25 min.',
+    id: 11,
+  },
+  {
+    image:
+      'https://i.pinimg.com/originals/61/3d/60/613d6095f19df359b80d0d824613a3e8.jpg',
+    title: ' Dark Chocolate Nut Clusters',
+    calories: '106 Calories',
+    category: 'Snacks',
+    duration: '15 min.',
+    id: 12,
+  },
+];
+export default function MealsPage() {
+  return (
+    <div>
+      <div className="flex justify-end mt-6 pr-32">
+        <DropDown />
```

Comment:
can we pass a function to the dropdown to get which item was clicked?
This can be useful for later filtering the meals